x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/ApiKeyServiceTests.java

```diff
@@ -778,39 +779,51 @@ public void testApiKeyCacheWillNotTraceLogOnEvictionDueToCacheTtl() throws Illeg
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/74586")
     public void testApiKeyAuthCacheWillLogWarningOnPossibleThrashing() throws Exception {
         ApiKeyService service = createApiKeyService(
-            Settings.builder().put("xpack.security.authc.api_key.cache.max_keys", 1).build());
+            Settings.builder().put("xpack.security.authc.api_key.cache.max_keys", 2).build());
         final Cache<String, ListenableFuture<CachedApiKeyHashResult>> apiKeyAuthCache = service.getApiKeyAuthCache();
 
         // Fill the cache
-        final String apiKeyId = randomAlphaOfLength(22);
-        apiKeyAuthCache.put(apiKeyId, new ListenableFuture<>());
+        apiKeyAuthCache.put(randomAlphaOfLength(20), new ListenableFuture<>());
+        apiKeyAuthCache.put(randomAlphaOfLength(21), new ListenableFuture<>());
         final Logger logger = LogManager.getLogger(ApiKeyService.class);
-        Loggers.setLevel(logger, Level.WARN);
+        Loggers.setLevel(logger, Level.TRACE);
         final MockLogAppender appender = new MockLogAppender();
         Loggers.addAppender(logger, appender);
         appender.start();
 
         try {
             // Prepare the warning logging to trigger
             service.getEvictionCounter().add(4500);
+            service.getLastEvictionCheckedAt().set(System.nanoTime() - 310L * 1_000_000_000L);
```

Comment:
Can you do something here to explain what that calculation is?
Possible options could be:
1. Extract to a local variable(s)
2. Use `TimeUnit.toNanos` to explain what `1_000_000_000L` means
3. A comment (if we can't solve it in other ways).

Reply:
I did both 1 and 3. Hopefully the change made the calculation less cryptic. In hindsight, I should have explained about the value of `310L` because it is rather arbitrary (300 seconds of checking interval plus 10 seconds buffer to counter flakiness)